From 17d73ad5bc1721f035f4605dabc5835818941734 Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Wed, 10 Dec 2014 19:42:48 +0800
Subject: [PATCH] media: rk30_camera_oneframe: fix link error

`.exit.text' referenced in section `.data.rel.local' of drivers/built-in.o: defined in discarded section `.exit.text' of drivers/built-in.o

Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 drivers/media/video/rk30_camera_oneframe.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/video/rk30_camera_oneframe.c b/drivers/media/video/rk30_camera_oneframe.c
index 5a7f870835d3..746203d85119 100755
--- a/drivers/media/video/rk30_camera_oneframe.c
+++ b/drivers/media/video/rk30_camera_oneframe.c
@@ -3257,7 +3257,7 @@ static int rk_camera_probe(struct platform_device *pdev)
     return err;
 }
 
-static int __exit rk_camera_remove(struct platform_device *pdev)
+static int rk_camera_remove(struct platform_device *pdev)
 {
     struct rk_camera_dev *pcdev = platform_get_drvdata(pdev);
     struct resource *res;
-- 
2.35.3

